### PR TITLE
demo documentation on how to install the plugin via helm

### PR DESCRIPTION
**Pull request by HerrmannHinz**
:)

### CODE REVIEW

File: demos/kubernetes-helm/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Jenkins can be installed in Kubernetes using [`helm`](https://github.com/helm/helm).
+The latest stable helm chart can be found [`here`](https://github.com/helm/charts/tree/master/stable/jenkins).
+
+To have the latest Version of JCASC installed we need to enable Jenkins to make use of the [`Experimental Update Center`](https://jenkins.io/doc/developer/publishing/releasing-experimental-updates/).
+
+The [`official docker image`](https://github.com/jenkinsci/docker/blob/master/Dockerfile#L60) provides enabling the Experimental Update Center via ENV vars:  
+
+```
+ENV JENKINS_UC https://updates.jenkins.io
```
[the remark this answers is not in the document]
Comment (casz) [reply demos/kubernetes-helm/README.md]: equal is not necessary it is just a different syntax

File: demos/kubernetes-helm/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+docker-workflow:1.17
+durable-task:1.22
+...
+```
+
+Done :)
+
+## Next?
+
+You can extend the Jenkins Helm Chart to make use of a Custom Config map.
```
[the remark this answers is not in the document]
Comment (julianxhokaxhiu) [reply demos/kubernetes-helm/README.md]: actually is should be `custom ConfigMap` :)

See https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/

File: demos/kubernetes-helm/README.md [hunk 3]
```diff
@@ -102,7 +101,7 @@ Done :)
 
 ## Next?
 
-You can extend the Jenkins Helm Chart to make use of a Custom Config map.
-This would enable you to mount in some additional files.
+You can extend the Jenkins Helm Chart to make use of a custom ConfigMap.
+This would enable you to mount in some additional files (like a )
```
Comment (casz) [inline demos/kubernetes-helm/README.md]: like a what? 🤔

File: demos/kubernetes-helm/README.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+```
+ENV JENKINS_UC https://updates.jenkins.io
+ENV JENKINS_UC_EXPERIMENTAL=https://updates.jenkins.io/experimental
+ENV JENKINS_INCREMENTALS_REPO_MIRROR=https://repo.jenkins-ci.org/incrementals
+```
+
+Now grab a copy of the helm chart [`values file`](https://github.com/helm/charts/blob/master/stable/jenkins/values.yaml) and adjust the Master part a little bit:
+
+```
```
[the remark this answers is not in the document]
Comment (HerrmannHinz) [reply demos/kubernetes-helm/README.md]: i updated this now.

File: demos/kubernetes-helm/README.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  # mind the last plugin in the list now ;)
+  InstallPlugins:
+      - kubernetes:latest
+      - kubernetes-credentials:latest
+      - workflow-aggregator:latest
+      - workflow-job:latest
+      - credentials-binding:latest
+      - git:latest
+      - configuration-as-code:0.10-alpha       
+...
```
[the remark this answers is not in the document]
Comment (HerrmannHinz) [reply demos/kubernetes-helm/README.md]: done

File: demos/kubernetes-helm/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (HerrmannHinz) [reply demos/kubernetes-helm/README.md]: done as well

File: demos/kubernetes-helm/README.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+      - configuration-as-code:0.10-alpha            
+```
+
+## Installations
+
+Now, deploy the Helm chart with those customized values:
+
+```
+helm install --name jenkins stable/jenkins \
+-f values.yml
```
Comment (julianxhokaxhiu) [inline demos/kubernetes-helm/README.md]: Can you make the whole snippet as one-line? It makes the Markdown file look better when scrolling.

I see at least no added value on keeping it in a separate line.

Comment (HerrmannHinz) [reply demos/kubernetes-helm/README.md]: done

**Pull request merged**